<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accountancy</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblOwners">
         <property name="text">
          <string>Owners:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vLayoutOwners"/>
       </item>
       <item>
        <widget class="Line" name="lineOwners">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblAccounts">
         <property name="text">
          <string>Accounts:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vLayoutAccounts"/>
       </item>
       <item>
        <widget class="Line" name="lineAccounts">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblReports">
         <property name="text">
          <string>Reports:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPieChart">
         <property name="text">
          <string>Pie chart</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBarChart">
         <property name="text">
          <string>Bar chart</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="lineSettings">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSettings">
         <property name="text">
          <string>Settings:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCategories">
         <property name="text">
          <string>Categories</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRules">
         <property name="text">
          <string>Rules</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>265</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="documentMode">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuAccounts">
    <property name="title">
     <string>Comptes</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionCredits"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <widget class="QMenu" name="menuTables">
     <property name="title">
      <string>Tables</string>
     </property>
     <addaction name="actionOwner"/>
     <addaction name="actionBanks"/>
     <addaction name="actionAccounts"/>
     <addaction name="actionTransactions"/>
     <addaction name="separator"/>
     <addaction name="actionCategories"/>
     <addaction name="actionSubcategories"/>
     <addaction name="actionBalanceNotes"/>
     <addaction name="actionRetailers"/>
    </widget>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="menuTables"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuAccounts"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport"/>
  </widget>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/import_icon&amp;24.png</normaloff>:/imgs/import_icon&amp;24.png</iconset>
   </property>
   <property name="text">
    <string>Importer...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionCredits">
   <property name="text">
    <string>Credits</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionOwner">
   <property name="text">
    <string>Owners</string>
   </property>
  </action>
  <action name="actionAccounts">
   <property name="text">
    <string>Accounts</string>
   </property>
  </action>
  <action name="actionTransactions">
   <property name="text">
    <string>Transactions</string>
   </property>
  </action>
  <action name="actionBanks">
   <property name="text">
    <string>Banks</string>
   </property>
  </action>
  <action name="actionCategories">
   <property name="text">
    <string>Categories</string>
   </property>
  </action>
  <action name="actionSubcategories">
   <property name="text">
    <string>Subcategories</string>
   </property>
  </action>
  <action name="actionBalanceNotes">
   <property name="text">
    <string>Balance Notes</string>
   </property>
  </action>
  <action name="actionRetailers">
   <property name="text">
    <string>Retailers</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
